src/nl/hannahsten/texifyidea/inspections/latex/redundancy/LatexMultipleIncludesInspection.kt

```diff
@@ -34,34 +32,33 @@ open class LatexMultipleIncludesInspection : TexifyInspectionBase() {
 
     override fun getDisplayName() = "Package has been imported multiple times"
 
-    override fun inspectFile(file: PsiFile, manager: InspectionManager, isOntheFly: Boolean): MutableList<ProblemDescriptor> {
-        val descriptors = descriptorList()
-
+    override fun inspectFile(file: PsiFile, manager: InspectionManager, isOntheFly: Boolean): List<ProblemDescriptor> {
         // Find all duplicates.
-        val packages = file.includedPackages(onlyDirectInclusions = true).map { it.name }
+        val packagesWithDuplicate = PackageUtils.getIncludedPackagesInFileset(file)
```

Comment:
I think this should now be changed to get only direct package inclusions, because now if you `\usepackage{graphicx}` this inspection fires because a package included by graphicx again includes graphicx. This inspection is here to warn if you do `\usepackage{graphicx}` twice in the same fileset.